### PR TITLE
[Fix] Clean Code for Better Readability

### PR DESCRIPTION
**Pull request by VILJkid**
# Introduced changes
- Early `return` wherever necessary
- Usage of `any` modern alias
- Limitation on named `return` wherever necessary
- Type assertion check

### CODE REVIEW

**Comment by tofers** [comment]
Thank you for your input. We will study and test the code

**Comment by lanyere** [comment]
The commit indeed has improvements with the code readability.

File: examples/main.go [hunk 1]
```diff
@@ -40,5 +40,6 @@ func handler(ctx *fiber.Ctx) error {
 	if err != nil {
 		return err
 	}
+
```
Comment (lanyere) [inline examples/main.go:43]: I'm not sure why, but ok :)

File: misc.go [hunk 2]
```diff
@@ -90,7 +91,7 @@ func statusChecker(status int) error {
 	}
 }
 
-func marshal(data interface{}, req *fiber.Request) (*fiber.Request, error) {
+func marshal(data any, req *fiber.Request) (*fiber.Request, error) {
```
Comment (lanyere) [inline misc.go:94]: *fiber.Request in the return is unnecessary.

**Review by lanyere — APPROVED** [review]
approved.

File: examples/main.go (hunk 1 above)
Comment (VILJkid) [reply examples/main.go:43]:
```go
s := "silence is an important part of music"
s = strings.Replace(s, "silence", "space/newline")
s = strings.Replace(s, "music", "code")
```
😉

File: misc.go (hunk 2 above)
Comment (VILJkid) [reply misc.go:94]: Yes, correct. Nice catch!

**Comment by VILJkid** [comment]
I'm glad you liked the updates.
Let's make code feel happy again!

**Pull request merged**